packages/babel-generator/src/buffer.js

```diff
@@ -101,7 +101,7 @@ export default class Buffer {
 
   rightBrace() {
     this.newline(true);
-    if (this.format.compact && !this._lastPrintedIsEmptyStatement) {
+    if (this.format.minified && !this._lastPrintedIsEmptyStatement) {
```

Comment:
How are we differentiating `compact` and `minified`? I was hoping the more minified mode would just be enabling the more dangerous stuff.

The `compact` description says:

> do not include superfluous whitespace characters and line terminators [true|false|auto]

which seems to include this